content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/troubleshooting-the-codeql-workflow.md

```diff
@@ -98,6 +98,8 @@ If an automatic build of code for a compiled language within your project fails,
 
 If your workflow fails with an error `No source code was seen during the build` or `The process '/opt/hostedtoolcache/CodeQL/0.0.0-20200630/x64/codeql/codeql' failed with exit code 32`, this indicates that {% data variables.product.prodname_codeql %} was unable to monitor your code. Several reasons can explain such a failure:
 
+1. There is no {% data variables.product.prodname_codeql %} supported source code in the repository. If this is the case, remove the {% data variables.product.prodname_codeql %} workflow.
```

Comment:
```suggestion
1. There is no source code in languages supported by {% data variables.product.prodname_codeql %} in the repository. If this is the case, remove the {% data variables.product.prodname_code_scanning %} workflow.
```

Maybe include a link to the list of currently supported languages?

Also elsewhere, the workflow is called the `code scanning` workflow, not the `codeql` workflow. Do we want the same here?

Reply:
Thanks for the review @aeisenberg. This article is specifically about troubleshooting for a workflow that calls the CodeQL action, so we should probably stick with CodeQL here. I'll make an amended version of your change and merge this.

```suggestion
1. There may be no source code in languages supported by {% data variables.product.prodname_codeql %} in the repository. Check the list of supported languages and, if this is the case, remove the {% data variables.product.prodname_codeql %} workflow. For more information, see "[About code scanning with CodeQL](/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning-with-codeql#about-codeql)
```